src/milia/api/io.cljs

```diff
@@ -46,10 +46,13 @@
 (defn token->headers
   "Builds request headers for the HTTP request by adding
   Authorization, X-CSRFToken and Cache-control headers where necessary"
-  [& {:keys [get-crsftoken? must-revalidate? accept-header]}]
+  [& {:keys [get-crsftoken? must-revalidate? accept-header auth-token]}]
```

Comment:
this is getting destructured as `auth-token`, but being set as `token`, https://github.com/onaio/milia/pull/146/files#diff-73181e15b22f8475f61619687e90b50dR68